packages/mdc-snackbar/foundation.js

```diff
@@ -65,6 +66,14 @@ export default class MDCSnackbarFoundation extends MDCFoundation {
     this.adapter_.deregisterActionClickHandler(this.actionClickHandler_);
   }
 
+  dismissesOnAction() {
```

Comment:
I'm running the demo and am getting an error when trying to show the snackbar. I think the reason is because the demo is trying to call `setDismissOnAction` directly on the component. As @traviskaufman mentioned, you would probably need a getter and setter in the component file to proxy in `setDismissOnAction` from the foundation (as well as `dismissesOnAction`).

Once that's in I think it looks good!

Thanks again for your help.